### ISSUE TITLE
Better Handling of Custom Emojis in Messages

### ISSUE THREAD

**Issue by xM8WVqaG** [RomainLK/ytc-filter#2]
## Problem

Currently when a custom emoji is sent in chat nothing is rendered in it's place in the filtered window.

## Expectation

Either the custom emoji should be rendered, or the alt text (which in this case, the `:word:` used to insert the emoji) of the image displayed.

## Reproduction

A moderator sends a message containing just a custom emoji at 0:11:14 in this video: https://youtu.be/jxy2KZ-GP_o?t=673

Screenshot of output:

![image](https://user-images.githubusercontent.com/44505331/91670769-5bcb1600-eb18-11ea-8db5-48127096e76a.png)

HTML in YouTube's chat pane:
```html
<div id="content" class="style-scope yt-live-chat-text-message-renderer">
  <span id="timestamp" class="style-scope yt-live-chat-text-message-renderer">11:14</span>
  <yt-live-chat-author-chip class="style-scope yt-live-chat-text-message-renderer">
    <!--css-build:shady-->
    <span id="author-name" dir="auto" class="moderator style-scope yt-live-chat-author-chip">
      くぅ。
      <span id="chip-badges" class="style-scope yt-live-chat-author-chip"></span>
    </span>
    <span id="chat-badges" class="style-scope yt-live-chat-author-chip">
      <yt-live-chat-author-badge-renderer class="style-scope yt-live-chat-author-chip" aria-label="Moderator" type="moderator" shared-tooltip-text="Moderator">
      <div id="image" class="style-scope yt-live-chat-author-badge-renderer">
        <yt-icon class="style-scope yt-live-chat-author-badge-renderer">
          <svg viewBox="0 0 16 16" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;" class="style-scope yt-icon">
            <g class="style-scope yt-icon">
              <path d="M9.64589146,7.05569719 C9.83346524,6.562372 9.93617022,6.02722257 9.93617022,5.46808511 C9.93617022,3.00042984 7.93574038,1 5.46808511,1 C4.90894765,1 4.37379823,1.10270499 3.88047304,1.29027875 L6.95744681,4.36725249 L4.36725255,6.95744681 L1.29027875,3.88047305 C1.10270498,4.37379824 1,4.90894766 1,5.46808511 C1,7.93574038 3.00042984,9.93617022 5.46808511,9.93617022 C6.02722256,9.93617022 6.56237198,9.83346524 7.05569716,9.64589147 L12.4098057,15 L15,12.4098057 L9.64589146,7.05569719 Z" class="style-scope yt-icon"></path>
            </g>
          </svg>
        </yt-icon>
      </div>
    </yt-live-chat-author-badge-renderer>
    <yt-live-chat-author-badge-renderer class="style-scope yt-live-chat-author-chip" aria-label="Member (10 months)" type="member" shared-tooltip-text="Member (10 months)">
      <div id="image" class="style-scope yt-live-chat-author-badge-renderer">
        <img src="https://yt3.ggpht.com/9LrrI_c6p7T6BT3OMK3IjxPJ_tWcjiQ_4PEIUi5GNVsSI1ay_sMBhpejUOLPJ-fQWTzfSRcV9g=s16-c-k" class="style-scope yt-live-chat-author-badge-renderer" alt="Member (10 months)">
      </div>
    </yt-live-chat-author-badge-renderer>
  </span>
  </yt-live-chat-author-chip>​
  <span id="message" dir="auto" class="style-scope yt-live-chat-text-message-renderer">
    <img class="emoji style-scope yt-live-chat-text-message-renderer" src="https://yt3.ggpht.com/lcuAO4tgAWUd7ZKhyTrXnjhDlRWt42KisrY1OlPai1qS2GvVNU80mH0C1ksaOOs2o8NlSqA-_g=w24-h24-c-k-nd" alt=":_pekoおーっ:" data-emoji-id="UC1DCedRgGHBdm81E1llLhOQ/q83XXZbMA8bf-QPU16voDA" shared-tooltip-text=":_pekoおーっ:" id="emoji-30">
  </span>
  <span id="deleted-state" class="style-scope yt-live-chat-text-message-renderer"></span>
  <a id="show-original" href="#" class="style-scope yt-live-chat-text-message-renderer"></a>
</div>
```
HTML in the filtered chat pane:
```html
<span class="vc-timestamp">11:14</span>
<span class="vc-author author">
  くぅ。
  <svg viewBox="0 0 16 16" preserveAspectRatio="xMidYMid meet" focusable="false" width="16" height="16" class="author-icon">
    <g>
      <path d="M9.64589146,7.05569719 C9.83346524,6.562372 9.93617022,6.02722257 9.93617022,5.46808511 C9.93617022,3.00042984 7.93574038,1 5.46808511,1 C4.90894765,1 4.37379823,1.10270499 3.88047304,1.29027875 L6.95744681,4.36725249 L4.36725255,6.95744681 L1.29027875,3.88047305 C1.10270498,4.37379824 1,4.90894766 1,5.46808511 C1,7.93574038 3.00042984,9.93617022 5.46808511,9.93617022 C6.02722256,9.93617022 6.56237198,9.83346524 7.05569716,9.64589147 L12.4098057,15 L15,12.4098057 L9.64589146,7.05569719 Z"></path>
    </g>
  </svg>
</span>
<span class="vc-message"></span>
```
For reference, payload from `get_live_chat_reply` endpoint:
```json
{
  "replayChatItemAction": {
    "actions": [
      {
        "addChatItemAction": {
          "item": {
            "liveChatTextMessageRenderer": {
              "message": {
                "runs": [
                  {
                    "emoji": {
                      "emojiId": "UC1DCedRgGHBdm81E1llLhOQ/q83XXZbMA8bf-QPU16voDA",
                      "shortcuts": [
                        ":_pekoおーっ:",
                        ":pekoおーっ:",
                        ":_おーっ:",
                        ":おーっ:"
                      ],
                      "searchTerms": [
                        "_pekoおーっ",
                        "pekoおーっ",
                        "_おーっ",
                        "おーっ"
                      ],
                      "image": {
                        "thumbnails": [
                          {
                            "url": "https://yt3.ggpht.com/lcuAO4tgAWUd7ZKhyTrXnjhDlRWt42KisrY1OlPai1qS2GvVNU80mH0C1ksaOOs2o8NlSqA-_g=w24-h24-c-k-nd",
                            "width": 24,
                            "height": 24
                          },
                          {
                            "url": "https://yt3.ggpht.com/lcuAO4tgAWUd7ZKhyTrXnjhDlRWt42KisrY1OlPai1qS2GvVNU80mH0C1ksaOOs2o8NlSqA-_g=w48-h48-c-k-nd",
                            "width": 48,
                            "height": 48
                          }
                        ],
                        "accessibility": {
                          "accessibilityData": {
                            "label": ":_pekoおーっ:"
                          }
                        }
                      },
                      "isCustomEmoji": true
                    }
                  }
                ]
              },
              "authorName": {
                "simpleText": "くぅ。"
              },
              "authorPhoto": {
                "thumbnails": [
                  {
                    "url": "https://yt3.ggpht.com/-P7JE6ljHYVU/AAAAAAAAAAI/AAAAAAAAAAA/bi8SRuNzqrg/s32-c-k-no-mo-rj-c0xffffff/photo.jpg",
                    "width": 32,
                    "height": 32
                  },
                  {
                    "url": "https://yt3.ggpht.com/-P7JE6ljHYVU/AAAAAAAAAAI/AAAAAAAAAAA/bi8SRuNzqrg/s64-c-k-no-mo-rj-c0xffffff/photo.jpg",
                    "width": 64,
                    "height": 64
                  }
                ]
              },
              "contextMenuEndpoint": {
                "commandMetadata": {
                  "webCommandMetadata": {
                    "ignoreNavigation": true
                  }
                },
                "liveChatItemContextMenuEndpoint": {
                  "params": "Q2pzS09Rb2FRMHRYV1RsbGVVSjNMWE5EUmxaSFRuZG5SV1F0ZVhORU5tY1NHME5PYldKcE5HbzRkM1Z6UTBaVk1FTXhVVzlrZEZJNFRXRjNPQkFBR2pnYURRb0xhbmg1TWt0YUxVZFFYMjhxSndvWVZVTXhSRU5sWkZKblIwaENaRzA0TVVVeGJHeE1hRTlSRWd0cWVIa3lTMW90UjFCZmJ5QUJLQVF5R2dvWVZVTldWbFpYYjBwd2NISjRTbmN4Y0RFNFpXRjFXazVu"
                }
              },
              "id": "CjkKGkNLV1k5ZXlCdy1zQ0ZWR053Z0VkLXlzRDZnEhtDTm1iaTRqOHd1c0NGVTBDMVFvZHRSOE1hdzg%3D",
              "timestampUsec": "1598793482849317",
              "authorBadges": [
                {
                  "liveChatAuthorBadgeRenderer": {
                    "icon": {
                      "iconType": "MODERATOR"
                    },
                    "tooltip": "Moderator",
                    "accessibility": {
                      "accessibilityData": {
                        "label": "Moderator"
                      }
                    }
                  }
                },
                {
                  "liveChatAuthorBadgeRenderer": {
                    "customThumbnail": {
                      "thumbnails": [
                        {
                          "url": "https://yt3.ggpht.com/9LrrI_c6p7T6BT3OMK3IjxPJ_tWcjiQ_4PEIUi5GNVsSI1ay_sMBhpejUOLPJ-fQWTzfSRcV9g=s16-c-k"
                        },
                        {
                          "url": "https://yt3.ggpht.com/9LrrI_c6p7T6BT3OMK3IjxPJ_tWcjiQ_4PEIUi5GNVsSI1ay_sMBhpejUOLPJ-fQWTzfSRcV9g=s32-c-k"
                        }
                      ]
                    },
                    "tooltip": "Member (10 months)",
                    "accessibility": {
                      "accessibilityData": {
                        "label": "Member (10 months)"
                      }
                    }
                  }
                }
              ],
              "authorExternalChannelId": "UCVVVWoJpprxJw1p18eauZNg",
              "contextMenuAccessibility": {
                "accessibilityData": {
                  "label": "Comment actions"
                }
              },
              "timestampText": {
                "simpleText": "11:14"
              }
            }
          },
          "clientId": "CNmbi4j8wusCFU0C1QodtR8Maw8"
        }
      }
    ],
    "videoOffsetTimeMsec": "674603"
  }
},
```

**Comment by RomainLK** [comment]
Thanks for all the info. I will try to have emote support for the next release.

**Comment by RomainLK** [comment]
Implemented in 1.4.0
